s3fs/tests/test_s3fs.py

```diff
@@ -661,3 +662,15 @@ def test_array(s3):
     with s3.open(a, 'rb') as f:
         out = f.read()
         assert out == b'A' * 1000
+
+
+def _get_s3_id(s3):
+        return id(s3.connect())
+
+
+pytest.importorskip('concurrent.futures')
```

Comment:
This line needs to be within the test.  If you move it down two lines and indent it all should be well